ansible/roles_ocp_workloads/ocp4_workload_shared_cluster_access/tasks/post_workload.yml

```diff
@@ -1,9 +1,32 @@
 ---
 # Implement your Post Workload deployment tasks here
-# --------------------------------------------------
+#--- --------------------------------------------------
+- name: Get console route
+  command: oc get route -n openshift-console console -o json
+  register: routeconsole
 
+- name: API for CLI
+  command: oc whoami --show-server
+  register: showserver
 
+- name: Set webconsole
+  set_fact:
+    webconsole: "http://{{ routeconsole.stdout | from_json | json_query('spec.host') }}"
+  when: showserver.changed
 
+- name: Print no SSH info
+  agnosticd_user_info:
+    msg: "{{ item }}"
+  loop:
+    - ""
+    - '**********'
```

Comment:
Did you test that this output showed correctly? I think they all need to be in double quotes, but I am not certain.

Reply:
Changed to double quotes.